### PR TITLE
ECOG and Karnofsky profiles

### PR DESCRIPTION
**Pull request by CarolinePotteiger**
Added ECOG and Karnofsky mCODE profiles

### CODE REVIEW

**Review by markkramerus — CHANGES_REQUESTED** [review]
I did an update... do a pull on your branch before making further changes. 

1.	ECOG/Karn: basedOn – should be Reference(ServiceRequest  | CarePlan)
2.	ECOG/Karn: Category: Should we have a fixed value instead of letting the user choose one? If so, what is the correct category for performance status? #procedure? #exam?
3.	ECOG/Karn: focus only Reference(Resource) – this is not a diff. Remove.
4.	ECOG/Karn: encounter – do not constrain to US Core
5.	ECOG/Karn: performer – constrain to Practitioner, not USCorePractitioner
6.	ECOG/Karn: derivedFrom --- do not constrain. Remove.
7.	ECOG/Karn: I don’t see any reason to constrain interpretation to 0..1 from 0..*. Remove.’
8.	ECOG/Karn: These are so similar, maybe we should create a parent class that has everything except the code and valueCodeableConcept binding. I put them into the same file for that reason. Make sure you update (pull) your branch.
9.	ECOG/Karn example: onsetDateTime does not belong. It is effectiveDateTime.
10.	ECOG/Karn example: asserter does not belong. It is practitioner.
11.	ECOG/Karn example: Do not set the focus. The focus is the patient (already given in the subject), not a specific condition.
12.	ECOG/Karn example: Should we set the category? If so, what is the correct value?  See comment #2.
13.	ECOG/Karn example: If the method is always going to be * method = SCT#5880005 "Clinical examination", should this added to the profiles?
14.	In the procedure example, we don’t need both a reasonReference and a reasonCode. One or the other.
15.	The treatment intent in procedure examples should use the extension name to refer to the relevant slice:
* extension[TreatmentIntent].valueCodeableConcept = SCT#373808002 "Curative - procedure intent"

**Comment by CarolinePotteiger** [comment]
I made the changes Mark suggested. 

For comment #2 and #12, in my mind, there could be multiple categories the physician used to determine the performance status such as procedure, laboratory, survey, and exam so I would think it's better to let the user choose one. However, I'm not exactly sure how physician's determine the status so I could be wrong. 

For comment #13, I picked the broad method of "clinical examination" because it encompasses multiple specific methods that could be used to determine these scores. However, the user may want to be specific in what methods he/she actually used.

File: PerformanceStatus.fsh [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+* dataAbsentReason 0..0
+* bodySite 0..0
+* specimen 0..0
+* device 0..0
+* referenceRange 0..0
+* hasMember 0..0
+* component 0..0
+* basedOn only Reference (ServiceRequest | CarePlan)
+* partOf only Reference (Procedure)
+* category only ObservationCategoryCodes (extensible)
```
Comment (markkramerus) [inline PerformanceStatus.fsh]: * category from ObservationCategoryCodes (extensible)

Comment (markkramerus) [reply PerformanceStatus.fsh]: Binding is always "from", not "only"

File: PerformanceStatus.fsh [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+* code = LNC#89245-5 "Karnofsky Performance Status [Interpretation]"
+* subject only Reference(Patient)
+* encounter only Reference(Encounter)
```
Comment (markkramerus) [inline PerformanceStatus.fsh]: Remove. Not sure why we constrain this.

File: PerformanceStatus.fsh [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+* dataAbsentReason 0..0
+* bodysite 0..0
+* specimen 0..0
+* device 0..0
+* referenceRange 0..0
+* hasMember 0..0
+* component 0..0
+* basedOn only Reference((ServiceRequest | CarePlan)
+* partOf only Reference(Procedure)
+* category only ObservationCategoryCodes (extensible)
```
Comment (markkramerus) [inline PerformanceStatus.fsh]: Same as above (from)

File: PerformanceStatus.fsh [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+* code = LNC#89262-0 "ECOG Performance Status [Interpretation]"
+* subject only Reference(Patient)
+* encounter only Reference(Encounter)
```
Comment (markkramerus) [inline PerformanceStatus.fsh]: Same as above (remove)

**Review by markkramerus — CHANGES_REQUESTED** [review]
See comments on individual lines.

**Review by markkramerus — APPROVED** [review]

**Pull request merged**